lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
Even it's only used by web app (actually there is only web app and the evaluator?), It's still make sense to let the gem to provide some API to let the web app declare languages and link to database IMO.

Declare in the gem also works, but the gem better provide some way to let it link to database (other than extensions) ?